### ISSUE TITLE
Support fallback to RelMeAuth for websites missing authorization endpoint

### ISSUE THREAD

**Issue by tantek** [aaronpk/Aperture#44]
When I try to sign-in I get an error message:

&gt; missing authorization endpoint
&gt;  Could not find your authorization endpoint

It would be great if instead it would fallback to supporting RelMeAuth perhaps via indielogin.com, so anyone who has already setup rel=me for signing into Indieweb.org could also sign-in to Aperture!

(Originally published at: http://tantek.com/2018/177/b1/support-fallback-relmeauth)

**Comment by aaronpk** [comment]
Unfortunately this won't really work. 

Even if you could sign in to Aperture, you still wouldn't be able to use any of the reader apps. The reader apps expect to be able to get an access token in order to make authenticated requests to Aperture. Getting an access token requires having an authorization endpoint and token endpoint.

If I allowed people to log in to Aperture without fully setting up IndieAuth, it would just be confusing because then they'd get an error trying to sign in to any apps. I think it's better to not mislead people at that stage, and require that they set up IndieAuth before being able to sign in to Aperture.

**Comment by EdwardHinkle** [comment]
I agree. I think the best stage here is to point them to some simple IndieAuth steps, kind of like Quill might. (Doesn’t have to be as dynamic as Quill). For Indigenous, I’ve attempted to do this by having links to the most popular IndieAuth software and services https://indigenous.abode.pub/ios/help/
